NOTICE

```diff
@@ -2,4 +2,11 @@ Ensembl
 Copyright 2020 EMBL-European Bioinformatics Institute
 
 This product includes software developed at:
-- EMBL-European Bioinformatics Institute
\ No newline at end of file
+- EMBL-European Bioinformatics Institute
+
+The following files are copyrighted to EMBL-EBI under the Creative Commons Attribution-ShareAlike 4.0 license. See https://creativecommons.org/licenses/by-sa/4.0/ for details of the license.
+
+- /src/ensembl/static/img/entity-viewer/icon_protein_annotations.svg
+- /src/ensembl/static/img/entity-viewer/icon_protein_interactions.svg
+- /src/ensembl/static/img/entity-viewer/icon_protein_ligands.svg
+- /src/ensembl/static/img/entity-viewer/icon_protein_structures.svg
```

Comment:
I've added the licence notice for the EMBL-EBI icons that we are using in the protein view.